analyses/cnv-chrom-plot/cn_status_heatmap.Rmd

```diff
@@ -420,14 +451,19 @@ heatmap <- ComplexHeatmap::Heatmap(
   bin_calls_mat,
   name = "CN status",
   col = color_key,
+  row_split = histologies$short_histology,
   cluster_columns = FALSE,
   cluster_rows = FALSE,
+  rect_gp = grid::gpar(col = "black", lwd = .0005),
   show_column_names = FALSE,
   show_row_names = FALSE,
+  row_labels = FALSE,
   bottom_annotation = chr_annot,
   right_annotation = hist_annot,
-  heatmap_legend_param = list(nrow = 1),
-  raster_quality = 8
+  heatmap_legend_param = list(nrow = 1, border = "black"),
+  raster_quality = 8, 
+  border = "black", 
```

Comment:
Changing this to `border=FALSE` may make the n=1 cases visible, at the expense of the pretty borders.

Reply:
Yes. It does. But the question is do we still even want the n=1 on this full version anyway?

Reply:
At this stage, I think we do want to include them. Alternatively, another quick fix might be to lump them into the "Other" group.

Reply:
I like that for now. I'll push a quick PR for that.